src/sysinfo.cc

```diff
@@ -328,9 +335,26 @@ double MyCPUUsage() {
     }
   }
   return MyCPUUsageRUsage();
+#else
+  HANDLE proc = GetCurrentProcess();
```

Comment:
it seems like changing MyCPUUsageRUsage to be this implementation for Windows is more accurate.